constants/labels.js

```diff
@@ -0,0 +1,70 @@
+
+const prefixes = {
+    CUSTOMER: 'Customer',
+    BUG: 'Bug',
+    FEATURE: 'Feature'
+};
+
+const bugTypes = {
+    PRODUCTION: 'production',
+    DEVELOPMENT: 'development'
+}
+
+const featureTypes = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+function getPrefix (label) {
+    const labelPrefix= label.split(':');
+    return (labelPrefix.length > 0) ? labelPrefix[0].trimRight() : '';
```

Comment:
Not strictly necessary as the `split` operator will return an array with one element if the separator character is not found i.e. it always returns an array. The thing to check for is that label is not null though - the split will have a problem with that.

Suggest:

`const getPrefix = (label) => (label || '').split(':')[0].trimRight();`

Reply:
Yes, indeed. It looks way better and simpler like that. Thanks!